tests/sdk_test/sdk_test.cpp

```diff
@@ -2110,9 +2116,193 @@ void MegaChatApiTest::TEST_Calls(unsigned int a1, unsigned int a2)
     TestChatRoomListener *chatroomListener = new TestChatRoomListener(this, megaChatApi, chatid);
 
     ASSERT_CHAT_TEST(megaChatApi[a1]->openChatRoom(chatid, chatroomListener), "Can't open chatRoom account 1");
+    ASSERT_CHAT_TEST(megaChatApi[a2]->openChatRoom(chatid, chatroomListener), "Can't open chatRoom account 2");
```

Comment:
Add a briefing description at the beginning of this function telling what this test does, as we have in the rest of tests.